src/Persistence.Tests/PaYaml/Serialization/PFxExpressionYamlConverterTests.cs

```diff
@@ -132,6 +132,44 @@ public void WriteYamlWithTrailingWhitespace()
         VerifySerialize("val1 & foo  \t", "'=val1 & foo  \t'");
     }
 
+    [TestMethod]
+    public void WriteYamlWithEscapedStringInCodeView()
+    {
+        using var _ = new AssertionScope();
+
+        VerifySerialize(
+        """" 
+        Set(gblFoo, true);
+        // The next line is completely blank:
+
+        Set(gblBar, 42);
+        """",
+        """"
+        |-
+          =Set(gblFoo, true);
+          // The next line is completely blank:
+
+          Set(gblBar, 42);
+        """");
+
+        // BUG: YamlDotNet doesn't serialize string when spaces are added in blank lines and shows escaped string in codeview
+        // LoadingPage.OnVisible with spaces on blank line in the middle, serialization adds \r\n for new lines on windows-os and \n for non-windows os
```

Comment:
Is this line is applicable to this test or the actual bug.

note: one caveot of the raw string `"""` is that the line returns might be inconsistent on different OSes, dependent on the build machine.
I would consider adding a `.Replace("\r\n", "\n")` or vice versa to the strings to ensure the inputs have the NewLine chars expected for the test.

Reply:
otherwise, you could always go back to using non-raw strings as the input, and be very explicit about the input.
I know this goes against my claim to try to use the raw string syntax. So use your own judgement on whether this is applicable.